applications/StructuralMechanicsApplication/python_scripts/Kratos_Main_Structural.py

```diff
@@ -1,110 +1,88 @@
 from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
 
-import KratosMultiphysics
-import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
-import KratosMultiphysics.ExternalSolversApplication as ExternalSolversApplication
-import os
-
-#### TIME MONITORING START ####
-
-# time control starts
-import time as timer
-print(timer.ctime())
-# measure process time
-t0p = timer.clock()
-# measure wall time
-t0w = timer.time()
-
-#
-def StartTimeMeasuring():
-    # measure process time
-    time_ip = timer.clock()
-    return time_ip
-
-def StopTimeMeasuring(time_ip, process, report):
-    # measure process time
-    time_fp = timer.clock()
-    if( report ):
-        used_time = time_fp - time_ip
-        print("::[KSM Simulation]:: [ %.2f" % round(used_time,2),"s", process," ] ")
-
-#### TIME MONITORING END ####
-
-#### PARSING THE PARAMETERS ####
-
-# Import define_output
+from KratosMultiphysics import *
```

Comment:
@philbucher In terms of absolute imports, we have internally some issues (see the discussion in #42). The point is that we don't want to impose on the user the need to know which application defines which variable (since it is an implementation detail and, in our experience, changes frequently), but this implies the absolute imports, which is less clear than the relative ones.
Until we find a satisfactory solution (i.e. everything except the variables is treated as relative imports, but this is a bit hard to achieve technically), I'd also recommend using absolute imports as in this PR.